README.md

```diff
@@ -9,6 +9,15 @@ This repository holds the source code for the Realm SDK for Flutter™ and Dart
 
 **This project is in the Beta stage. The API should be quite stable, but occasional breaking changes may be made.**
 
+## Features
+
+* **Mobile-first:** Realm is the first database built from the ground up to run directly inside phones, tablets, and wearables.
+* **Simple:** Data is directly [exposed as objects](https://docs.mongodb.com/realm/dotnet/objects/) and [queryable by code](https://docs.mongodb.com/realm/dotnet/query-engine/), removing the need for ORM's riddled with performance & maintenance issues. Plus, we've worked hard to [keep our API down to just a few common classes](https://pub.dev/documentation/realm/latest/): most of our users pick it up intuitively, getting simple apps up & running in minutes.
```

Comment:
```suggestion
* **Simple:** Data is directly [exposed as objects](https://www.mongodb.com/docs/realm/sdk/flutter/realm-database/define-realm-object-schema/) and [queryable by code](https://www.mongodb.com/docs/realm/sdk/flutter/realm-database/read-and-write-data/), removing the need for ORM's riddled with performance & maintenance issues. Plus, we've worked hard to [keep our API down to just a few common classes](https://pub.dev/documentation/realm/latest/): most of our users pick it up intuitively, getting simple apps up & running in minutes.
```

Reply:
that url is not the same so I suggest we keep it to .NET until we have the correct section in the Flutter docs https://www.mongodb.com/docs/realm/sdk/flutter/realm-database/define-realm-object-schema/

Reply:
I think these two links have the same meaning. https://www.mongodb.com/docs/realm/sdk/flutter/realm-database/define-realm-object-schema/ and https://www.mongodb.com/docs/realm/sdk/flutter/realm-database/read-and-write-data/
Do we expect to have the same pages as .NEt SDK?

Reply:
They do not have similar meaning. The contents is very different. And yes we should have these fundamental pages documented for Flutter as well.

Reply:
OK

Reply:
I think we can as well remove the links and leave it as text. I think linking to the .NET docs here will be more confusing than helpful.